Sources/SwiftGRPC/Core/Server.swift

```diff
@@ -50,6 +50,7 @@ public class Server {
   /// - Parameter address: the address where the server will listen
   /// - Parameter key: the private key for the server's certificates
   /// - Parameter certs: the server's certificates
+  /// - Parameter rootCerts: for client validation and will enforce client validation when provided
```

Comment:
super-nit: "used to validate client certificates; will enable enforcing valid client certificates when provided"

Reply:
Fixed 😄